*** Starting uWSGI 2.0.18 (64bit) on [Mon May  4 16:08:36 2020] ***
compiled with version: 8.3.1 20190507 (Red Hat 8.3.1-4) on 04 May 2020 07:45:51
os: Linux-4.18.0-147.8.1.el8_1.x86_64 #1 SMP Thu Apr 9 13:49:54 UTC 2020
nodename: soda
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/projects/Django-Hexo-Matery
writing pidfile to uwsgi.pid
detected binary path: /home/projects/Django-Hexo-Matery/env/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/projects/Django-Hexo-Matery
your processes number limit is 7196
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8080 fd 7
uwsgi socket 0 bound to TCP address 127.0.0.1:37761 (port auto-assigned) fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.3 (default, Apr 29 2020, 22:37:16)  [GCC 8.3.1 20190507 (Red Hat 8.3.1-4)]
Python main interpreter initialized at 0x1765f10
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 4 seconds on interpreter 0x1765f10 pid: 3606 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3606)
spawned uWSGI worker 1 (pid: 3607, cores: 2)
spawned uWSGI worker 2 (pid: 3608, cores: 2)
spawned uWSGI worker 3 (pid: 3609, cores: 2)
spawned uWSGI worker 4 (pid: 3610, cores: 2)
spawned uWSGI http 1 (pid: 3611)
*** Starting uWSGI 2.0.18 (64bit) on [Mon May  4 17:41:29 2020] ***
compiled with version: 8.3.1 20190507 (Red Hat 8.3.1-4) on 04 May 2020 07:45:51
os: Linux-4.18.0-147.8.1.el8_1.x86_64 #1 SMP Thu Apr 9 13:49:54 UTC 2020
nodename: soda
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/projects/Django-Hexo-Matery
writing pidfile to uwsgi.pid
detected binary path: /home/projects/Django-Hexo-Matery/env/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/projects/Django-Hexo-Matery
your processes number limit is 7196
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8080).
bind(): Address already in use [core/socket.c line 769]
